### ISSUE TITLE
Job cancelled: Access violation

### ISSUE THREAD

**Issue by tmuelli** [neodynamic/JSPrintManager#19]
Hi there,
I get an error message after allowing eval print dialog from JSPM. I am trying to print an PDF-File, which was converted to blob before. 
Here is my code:
```
if (JSPM.JSPrintManager.websocket_status == JSPM.WSStatus.Open) {
  var cpj = new JSPM.ClientPrintJob();
  cpj.clientPrinter = new JSPM.DefaultPrinter();
  var file = new JSPM.PrintFilePDF(response.body, JSPM.FileSourceType.BLOB, newFileName, 1);
  cpj.files.push(file);
  cpj.sendToClient();
}
```

I've already checked `response.body`. It's got a valid size and type `application/pdf`.
Any guesses why this is happening?

Thanks in advance.

**Comment by neodynamic** [comment]
Please provide us a complete source code for reproducing the issue AND tell us what exact version of our product you're using. Thanks,

**Comment by tmuelli** [comment]
Hi,

your answer reminded me to check if I was using the latest version. The issue resolved after upgrading JSPM from v3.0.4 to v3.0.5. 

Thanks!
